src/WallsComputation.cpp

```diff
@@ -72,7 +72,7 @@ void WallsComputation::generateWalls(SliceLayerPart* part, SectionType section_t
         part->wall_toolpaths = wall_tool_paths.getToolPaths();
         part->inner_area = wall_tool_paths.getInnerContour();
     }
-    part->print_outline = part->outline;
+    part->print_outline = Simplify(settings).polygon(part->outline);
```

Comment:
I'm not super sure about this simplify; later in the pipeline we already simplify these same polygons

https://github.com/Ultimaker/CuraEngine/blob/7b90564857e4da887844b73ea7f53eb1fd5a7adc/src/WallToolPaths.cpp#L74

https://github.com/Ultimaker/CuraEngine/blob/7b90564857e4da887844b73ea7f53eb1fd5a7adc/src/WallsComputation.cpp#L116

We are simplifying the same polygon in multiple places. This doesn't hurt per-se but it does give me the feeling of being less in control of the process.

Reply:
I don't think we should change this now, but I think we should be aware in the future that these simplify functions are executed in multiple places, and removing one simplify doesn't un-simplify the per-se?

Reply:
Agreed. I put this here just to be sure to be honest.